### DISCUSSION TITLE
How to use IDM to take over download tasks?

### DISCUSSION THREAD

**Discussion by Sirius0060** [Eloston/ungoogled-chromium#2666] [Q&A]
Even installing the IDM extension cannot take over.

**Comment by opzch** [comment]
Ensure that IDM is running and file extension is on IDM Options -> File types tab. If didn't work:
- Remove the IDM extension, temporarily disable "always-on incognito", close all running browsers and background processes
- IDM Options -> General -> Use advanced browser integration -> Uncheck; close the survey
- Restart IDM and enable integration again
- Click "Add browser"... and point to the chrome.exe file (not chrlauncher or other shortcuts).
It should then appear as "Chromium" or "Google Chrome" in the list, with the checkbox active.
Launch Chromium, and if it works correctly, you should see that "IDM Integration Module was installed" (and if you create a new browser profile, it should get automatically installed too); enable the extension and allow it to run in incognito if you use always-on incognito mode.
